ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/inmemory/InMemoryLedger.scala

```diff
@@ -87,13 +88,13 @@ class InMemoryLedger(
   // need to take the lock to make sure the two pieces of data are consistent.
   override def snapshot(): Future[LedgerSnapshot] =
     Future.successful(this.synchronized {
-      LedgerSnapshot(entries.ledgerEnd, Source(acs.contracts))
+      LedgerSnapshot(entries.ledgerEnd, Source[ActiveContract](acs.activeContracts.map(_._2)))
```

Comment:
```suggestion
      LedgerSnapshot(entries.ledgerEnd, Source[ActiveContract](acs.activeContracts.values))
```

Reply:
Now I remember why I had this expression: `acs.activeContracts.values` is a mutable iterable (even though the map is immutable), but `Source` takes an immutable iterable. How do I best convert these? `acs.activeContracts.values.toList`?

Reply:
Yikes, that's disappointing of the scala stdlib api. Let's keep it as is.

Reply:
Changed to `Source.fromIterator(() => acs.activeContracts.valuesIterator))`